ssz-rs/src/merkleization/multiproofs.rs

```diff
@@ -52,7 +52,7 @@ pub fn calculate_merkle_root(
 ) -> Result<Node, Error> {
     let path_length = get_path_length(index)?;
     if path_length != proof.len() {
-        return Err(Error::InvalidProof)
+        return Err(Error::InvalidProof);
```

Comment:
this repo prefers a nightly lint option that doesn't require trailing `;` here
```suggestion
        return Err(Error::InvalidProof)
```